Gopkg.lock

```diff
@@ -0,0 +1,105 @@
+# This file is autogenerated, do not edit; changes may be undone by the next 'dep ensure'.
+
+
+[[projects]]
+  branch = "master"
+  name = "github.com/agl/ed25519"
+  packages = [".","edwards25519"]
+  revision = "5312a61534124124185d41f09206b9fef1d88403"
```

Comment:
This does not match the current revision being used.  In order to update dependencies, all commits needs to be review first.